The tvOS build error is known: https://webkit.slack.com/archives/CU64U6FDW/p1758389552029759

```
/Volumes/Data/worker/tvOS-18-Build-EWS/build/WebKitBuild/Release-appletvos/DerivedSources/WebCore/XMLViewerCSS.h:4:29: error: use of undeclared identifier 'Latin1Character'
    4 | static constexpr std::array<Latin1Character, 930> XMLViewer_css
      |                             ^
In file included from /Volumes/Data/worker/tvOS-18-Build-EWS/build/WebKitBuild/Release-appletvos/DerivedSources/WebCore/unified-sources/UnifiedSource491.cpp:2:
In file included from /Volumes/Data/worker/tvOS-18-Build-EWS/build/Source/WebCore/xml/XMLTreeViewer.cpp:48:
/Volumes/Data/worker/tvOS-18-Build-EWS/build/WebKitBuild/Release-appletvos/DerivedSources/WebCore/XMLViewerJS.h:4:29: error: use of undeclared identifier 'Latin1Character'
    4 | static constexpr std::array<Latin1Character, 9767> XMLViewer_js
      |                             ^
2 errors generated.
```

--- a/Source/WebCore/PAL/pal/spi/cocoa/QuartzCoreSPI.h
+++ b/Source/WebCore/PAL/pal/spi/cocoa/QuartzCoreSPI.h
@@ -233,6 +233,9 @@ typedef enum {
 @interface CARemoteEffect: NSObject <NSCopying, NSSecureCoding>
 @end
 
+@interface CARemoteExternalEffect: CARemoteEffect
+@end
+
 @interface CARemoteEffectGroup : CARemoteEffect
 + (instancetype)groupWithEffects:(NSArray<CARemoteEffect *> *)effects;
 @property (copy) NSString *groupName;
@@ -361,3 +364,8 @@ extern NSString * const kCASnapshotOriginY;
 extern NSString * const kCASnapshotTransform;
 extern NSString * const kCASnapshotTimeOffset;
 #endif
+
+// FIXME:rdar://160881286 - clean-up once in SDK.
+@interface CARemoteExternalEffect (Radar_160881286)
++ (instancetype)rcp_requiresTwoHandedInteractionProcessorEffect;
+@end

--- a/Source/WebKit/Platform/spi/ios/UIKitSPI.h
+++ b/Source/WebKit/Platform/spi/ios/UIKitSPI.h
@@ -639,6 +639,9 @@ extern NSString * const UIPresentationControllerDismissalTransitionDidEndComplet
 @interface UIActivityViewController ()
 @property (nonatomic) BOOL allowsCustomPresentationStyle;
 @end
+@interface UIView ()
+- (void)_requestRemoteEffects:(NSArray *)effects forKey:(NSString *)key;
+@end
 #endif // PLATFORM(VISION)
 
 @interface UIView ()

--- a/Source/WebKit/UIProcess/API/ios/WKWebViewIOS.mm
+++ b/Source/WebKit/UIProcess/API/ios/WKWebViewIOS.mm
@@ -224,6 +224,11 @@ static WebCore::IntDegrees deviceOrientationForUIInterfaceOrientation(UIInterfac
     [_contentView setFrame:bounds];
     [_scrollView addSubview:_contentView.get()];
     [_scrollView addSubview:[_contentView unscaledView]];
+
+#if PLATFORM(VISION)
+    if ([[CARemoteExternalEffect class] respondsToSelector:@selector(rcp_requiresTwoHandedInteractionProcessorEffect)])
+        [_scrollView _requestRemoteEffects:@[[CARemoteExternalEffect rcp_requiresTwoHandedInteractionProcessorEffect]] forKey:@"input"];
+#endif
 }
 
 - (void)_setObscuredInsetsInternal:(UIEdgeInsets)obscuredInsets

